src/library/scala/collection/convert/Wrappers.scala

```diff
@@ -188,11 +188,21 @@ private[collection] trait Wrappers {
           val (k, v) = ui.next()
           prev = Some(k)
           new ju.Map.Entry[A, B] {
-            import scala.util.hashing.byteswap32
             def getKey = k
             def getValue = v
             def setValue(v1 : B) = self.put(k, v1)
-            override def hashCode = byteswap32(k.##) + (byteswap32(v.##) << 16)
+
+            /**
+             * It's important that this implementation conform to the contract
```

Comment:
OK -- what's preferred:

```
// this kind
// of comment
```

or 

```
/* This
  * kind
  */
```

Reply:
doesn't matter, I don't think we have a consistent style on that. personally, I tend to like `//` because it makes it easier to see that something definitely isn't a Scaladoc comment.

Reply:
OK, will do, thanks